.config.sample

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file is not supposed to be committed now. It will be part of another issue.